solutions/USACO jan19-icy-perimeter.mdx

```diff
@@ -0,0 +1,93 @@
+---
+id: usaco-895
+source: USACO Silver 2019 January
+title: Icy Perimeter
+author: Tanish Tyagi 
+---
+
+[Official Analysis](http://www.usaco.org/current/data/sol_perimeter_silver_jan19.html)
+
+<LanguageSection>
+
+<CPPSection>
+
+## C++ Implementation
+
+```cpp
+//created by Tanish Tyagi
+#include <bits/stdc++.h>
+
+using namespace std;
+
+#define ll long long
+#define MAXN 1005
+#define MAXM 1005
```

Comment:
Prefer `using ll = long long` and `const int MAXN = 1005, MAXM = 1005;` as noted here:

https://usaco.guide/general/generic/#macros

(also, is it necessary to have separate variables for `MAXN` and `MAXM` when the grid is square?)

Reply:
please address this

Reply:
actually, I think you did but the changes were reverted?